introduction/zh/提纲.asciidoc

```diff
@@ -0,0 +1,50 @@
+:doctype: book
```

Comment:
What English file was the base for this file?

Reply:
https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/introduction/outline.asciidoc   check this Russ

Reply:
Got it - thank you!